src/Main.js

```diff
@@ -8,30 +8,39 @@ class Main extends React.Component {
       showBeast,
       removeBeastFilter,
       resetBeastFilter,
+      setHornLevel,
+      byHorns,
       filterBeasts,
       filters,
       displayedBeasts,
     } = this.props;
+    // I know, bad naming, but this takes horns into consideration.
+    const finalDisplayedBeasts = !byHorns ?
+      displayedBeasts
+      : displayedBeasts.filter(beast => {
+      return beast.horns === byHorns
+    })
     return (
       <>
         <BeastSearcher
           filters={filters}
           filterBeasts={filterBeasts}
           resetBeastFilter={resetBeastFilter}
-          removeBeastFilter={removeBeastFilter} />
-        {!displayedBeasts.length && (
-          <h3 className="no-results">No results with those search filters, sorry!</h3>
+          removeBeastFilter={removeBeastFilter}
+          setHornLevel={setHornLevel}
+          byHorns={byHorns} />
+        {!finalDisplayedBeasts.length && (
+          <h3 className="no-results">No results with those search conditions, sorry!</h3>
         )}
         <div className="beast-flex">
-          {!!displayedBeasts.length && (
-            displayedBeasts.map((beastData, index) => (
-              <HornedBeast
-                key={index}
-                showBeast={showBeast}
-                title={beastData.title}
-                description={beastData.description}
-                imageUrl={beastData.image_url}
-                horns={beastData.horns} />
+          {!!finalDisplayedBeasts.length && (finalDisplayedBeasts.map((beastData, index) => (
```

Comment:
So, that double "!!" is unecessary. If it's "not" "not" than it just is. finalDisplayedBeasts works on it's own. No double bang needed.